src/13_file_io.py

```diff
@@ -10,9 +10,19 @@
 
 # YOUR CODE HERE
 
+filename = "foo.txt"
+files = open(filename, "r")
```

Comment:
trying writing this line the same way your wrote line 25.

Reply:
Thanks! I figured it out. I needed to add the file path in order for Python to read it. Also changed the bar.txt code since it wasn't writing on separate lines.